FirebaseTestingSupport/Firestore/Sources/FIRQueryFake.mm

```diff
@@ -14,8 +14,6 @@
 
 #import "FirebaseTestingSupport/Firestore/Sources/Public/FirebaseFirestoreTestingSupport/FIRQueryFake.h"
 
-#import "Firestore/Source/API/FIRQuery+Internal.h"
```

Comment:
This is test code and the change was necessary to get FirestoreTestingSupport target building.